### PR TITLE
Update manifest.yaml

### PR DESCRIPTION
**Pull request by Kostov6**
**What this PR does / why we need it**:
Absolute to relative path conversion in manifest for simplification

**Special notes for your reviewer**:
None

**Release note**:

```doc dependency
Updated [docforge](https://github.com/gardener/docforge) version and simplified path transformations in docforge manifest
```

### CODE REVIEW

**Review by holgerkoser — APPROVED** [review]

File: .docforge/manifest.yaml [hunk 1]
```diff
@@ -1,49 +1,38 @@
-{{- $vers := Split .versions "," -}}
-{{ $defaultBranch := (index $vers 0) }}
 structure:
 - name: _index.md
-  source: https://github.com/gardener/dashboard/blob/{{$defaultBranch}}/README.md
+  source: /README.md
```
Comment (holgerkoser) [inline .docforge/manifest.yaml:3]: It seems that the step `concourse-ci/check-docforge` cannot handle this short relative urls.

Comment (Kostov6) [reply .docforge/manifest.yaml:3]: Recently we have updated docforge that now handles relative paths

**Review by Kristian-ZH — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**